### PR TITLE
Escape  params, fix #52

### LINKED ISSUE #52: Security Vulnerability - XSS
**About**

Hi, by using your design library, I came upon an **XSS** ( Cross-Site Scripting ) vulnerability. The issue occurs at the ?cat= parameter. An attacker could insert an XSS payload as the value of the parameter and code would be executed.

The exploit would look like this :
`https://design.inrupt.com/atomic-core/?cat="><svg/onload=confirm(document.domain)>`
( This only executes a PopUp alert saying the domain, so don't worry)

This means everyone that is using your library is vulnerable to this security issue.
This also means, that your website is vulnerable to this issue as you can confirm by going into the URL above ( Use **Firefox browser** for this, Google Chrome as protection against XSS )

**Fix**
Sanitize user input when reflecting on the page. Such as using `htmlspecialchars();` PHP function every time we are reflecting user input is a good habit to prevent XSS. 

Thank you for your time, Castilho.

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
You can test this on https://solid-sdk-style-guide.herokuapp.com/atomic-core/?cat=%22%3E%3Csvg/onload=confirm(document.domain)%3E

File: atomic-core/copy.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ function resizeIframe(obj) {
         </li>
 
 
-    <?php } ?>
+    <?php }) ?>
```
Comment (michielbdejong) [inline atomic-core/copy.php]: Unrelated fix, my editor was complaining about this. I don't think this page is actually used in practice.

**Comment by james-martin-jd** [comment]
LGTM, seems like a sound solution

**Comment by michielbdejong** [comment]
Great! @tony-morelli can you merge and deploy?

**Pull request merged**